weblate/utils/search.py

```diff
@@ -265,6 +265,8 @@ def has_sql(text):
         return Q(shaping__isnull=False)
     if text == "label":
         return Q(labels__isnull=False)
+    if text == "context":
+        return Q(context__isnull=False)
```

Comment:
The context field can not be null, it can be empty string.

Reply:
Fixed.